datageneration/property_generator.py

```diff
@@ -47,6 +47,7 @@ def generate_numerical_property(self, tag_property: TagProperty) -> Property:
         if tag.key == "height":
             # todo rename this
             generated_numerical_value = get_random_decimal_with_metric(max_digits=5)
+            generated_numerical_value = f'{generated_numerical_value.magnitude} {generated_numerical_value.metric}'
```

Comment:
From what I saw, the "rewrite_distance" method to remove whitespaces at random is only applied to relation distances.. Should we maybe do the same for height? We can also try it like this if that is too much work.

Reply:
it is fixed in gpt_generation.